FlorenceBML/BMLacq812/BMLacq812.xml

```diff
@@ -72,6 +244,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
         <revisionDesc>
```

Comment:
add Geez to langusage

Reply:
Done.

Reply:
Done.

Reply:
But I cannot see the full images but rather the small item on the bottom line only. It does not work properly in neither in Chrome nor in firefox. Maybe I should shut down the computer and try a new start.

Reply:
I see it now too, it is very strange, as other manuscripts seem to work, I will try to explore when I am in office what the bug is with this MS